DOL.WHD.Section14c.Test/RepositoryMocks/OrganizationRepositoryMock.cs

```diff
@@ -0,0 +1,84 @@
+﻿using DOL.WHD.Section14c.DataAccess;
+using DOL.WHD.Section14c.Domain.Models;
+using DOL.WHD.Section14c.Domain.Models.Submission;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace DOL.WHD.Section14c.Test.RepositoryMocks
+{
+    public class OrganizationRepositoryMock: IOrganizationRepository
```

Comment:
I've asked before and never got an answer, but I'm going to ask more firmly this time: why are there so many custom mock classes when the project is already using Moq?  These mocks implement logic and so they introduce uncertainty into the tests.  That is, if a test fails, the tests are sufficiently complex that you can't be sure whether the code under test has broken or if the *tests* have broken.

Please consider using Moq, but in the meantime, please justify these custom mock classes.

Reply:
@mgwalker I created those custom mock classes to provides test data that I will be able to compare with during testing. They helped me corrected a few implementation errors. Good points on the custom mock classes could introduce complexity ; I will try to use Moq as much as possible.

Reply:
Gotcha.  Works for me.  Thanks for humoring me.  :)